client/src/styles/body/_body.scss

```
@@ -0,0 +1,134 @@
+$margin: 16px;
+$list-width: 308px;
+
+main {
+  display: flex;
+  padding: 0 80px;
+}
+
+.column {
+  display: flex;
+  flex-direction: column;
+  width: $list-width;
+  margin-right: $margin;
+}
+
+.column-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  width: 100%;
+  margin-bottom: $margin;
+
+  .column-title {
+    @include font_style(700, 18px, 26px);
+    display: inline-block;
+    padding-right: 8px;
+  }
+
+  .item-count {
+    @include size(24px, 24px);
+    @include font_style(700, 14px, 20px);
+    display: inline-block;
+    border-radius: 50%;
+    background-color: $gray4-color;
+    text-align: center;
+    vertical-align: bottom;
+  }
+
+  .plus-icon {
+    margin-right: 12px;
+
+    path {
+      fill: $gray4-color;
+    }
+  }
+
+  .plus-icon:hover {
+    path {
+      fill: $blue-color;
+    }
+  }
+
+  .close-icon {
+    path {
+      fill: $gray4-color;
+    }
+  }
+
+  .close-icon:hover {
+    path {
+      fill: $red-color;
+    }
+  }
+}
+
+.column-list {
+  .list_item {
+    display: flex;
+    align-items: flex-start;
+    margin-bottom: $margin;
+    padding: 16px;
+    border-radius: 6px;
+  }
+
+  .item-info {
+    width: 100%;
+
+    input {
+      width: 100%;
+      border: none;
+      background-color: none;
+    }
+
+    input:focus {
+      outline: none;
+    }
```

여기도 셀렉터의 하위 요소들을 내부로 들여올 수 있겠네요~

```suggestion
    input {
      width: 100%;
      border: none;
      background-color: none;
      &:focus {
        outline: none;
      }
    }
```